cfgov/jinja2/v1/_layouts/base-base.html

```diff
@@ -0,0 +1,245 @@
+<!--[if IE 7]>         <html lang="es" class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html lang="en" class="no-js lt-ie10 lt-ie9"> <![endif]-->
+<!--[if IE 9]>         <html lang="en" class="no-js lt-ie10"> <![endif]-->
+<!--[if gt IE 8]><!--> <html lang="en" class="no-js"> <!--<![endif]-->
+
+<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# {% block og_article_prefix %}{% endblock %}">
+
+{#
+    ===========
+    GLOBAL META
+    ===========
+#}
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8">
+    {% block meta_viewport %}
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% endblock meta_viewport %}
+
+{#
+    ==================
+    PAGE-SPECIFIC META
+    ==================
+#}
+
+    <title>
+        {%- block title -%}
+            {%- if page -%}
+                {{ page.seo_title or page.title }} | Consumer Financial Protection Bureau
+            {%- else -%}
+                Consumer Financial Protection Bureau
+            {%- endif %}
+        {%- endblock title -%}
+    </title>
+    <meta name="description"
+          content="
+            {%- block desc -%}
+                {{- page.search_description if page and page.search_description
+                    else 'Our vision is a consumer finance marketplace that works for American consumers, responsible providers, and the economy as a whole.'
+                -}}
+            {%- endblock -%}
+          ">
+    <link rel="canonical" href="{{ request.url | lower }}">
+
+{# Open Graph properties #}
+    {# Required Open Graph properties #}
+    <meta property="og:title" content="{% block og_title %}{{ self.title() }}{% endblock %}">
+    <meta property="og:type" content="{% block og_type %}website{% endblock %}">
+    <meta property="og:url" content="{{ request.url }}">
+{% block og_image %}
+{% if page and page.meta_image %}
+    {% set meta_image_url = request.build_absolute_uri(image(page.meta_image, 'original').url) %}
+    <meta property="og:image" content="{{ meta_image_url }}">
+    <meta property="twitter:image" content="{{ meta_image_url }}">
+    {% if page.meta_image.should_display_summary_large_image %}
+    <meta name="twitter:card" content="summary_large_image">
+    {% endif %}
+{% else %}
+    <meta property="og:image"
+          content="{{ request.scheme }}://{{ request.get_host() }}{{ static('img/logo_open-graph_facebook.png') }}">
+    <meta property="twitter:image"
+          content="{{ request.scheme }}://{{ request.get_host() }}{{ static('img/logo_open-graph_twitter.png') }}">
+{% endif %}
+{% endblock og_image %}
+    {# Optional Open Graph properties #}
+    <meta property="og:description"
+          content="
+              {%- block og_desc -%}
+                  {%- if page -%}
+                      {{- page.search_description or page.seo_title or page.title -}}
+                  {%- endif -%}
+              {%- endblock -%}
```

Comment:
Please update this to `{% endblock og_desc %}` for consistency with most of the other non-inline `endblock` tags in this template.